example/lib/main.dart

```diff
@@ -45,8 +46,11 @@ class _MyAppState extends State<MyApp> {
       print('startRecorder: $path');
```

Comment:
Hope you can remove this.

Reply:
This line already exist in the source, Hopes it will be helpful for the debug purpose. So not removed.